WordPress/src/main/java/org/wordpress/android/ui/media/MediaPreviewActivity.java

```diff
@@ -512,6 +513,28 @@ private void setLookClosable(boolean lookClosable) {
         }
     }
 
+    private void shareMedia() {
+        MediaModel media = mMediaStore.getMediaWithLocalId(mMediaId);
+        if (media == null) {
+            ToastUtils.showToast(this, R.string.error_media_not_found);
+            return;
+        }
+
+        Intent intent = new Intent(Intent.ACTION_SEND);
+        intent.setType("text/plain");
+        intent.putExtra(Intent.EXTRA_TEXT, media.getUrl());
+        if (!TextUtils.isEmpty(media.getTitle())) {
+            intent.putExtra(Intent.EXTRA_SUBJECT, media.getTitle());
+        } else if (!TextUtils.isEmpty(media.getDescription())) {
+            intent.putExtra(Intent.EXTRA_SUBJECT, media.getDescription());
+        }
+        try {
+            startActivity(Intent.createChooser(intent, getString(R.string.reader_share_link)));
```

Comment:
It might be better to use the `R.string.share_link` here instead of the specific `R.string.reader_share_link`. Better yet, if we don't think the localized version would be special for reader sharing, we can just remove the `reader_share_link` resource all together. What do you think?